.github/workflows/bump.yaml

```diff
@@ -0,0 +1,32 @@
+name: Weekly npins update
+
+on:
+  schedule:
+    # run at 00:00 UTC every Monday
+    - cron: "0 0 * * 1"
+  # allow manual trigger
+  workflow_dispatch:
+
+permissions:
```

Comment:
```suggestion
# NOTE: for this to work you need to set "Workflow permissions" to "Read and write permissions"
# and check "Allow GitHub Actions to create and approve pull requests" in
# https://github.com/organizations/<owner>/settings/actions
# in order enable the checkbox "Allow GitHub Actions to create and approve pull requests" in
#  https://github.com/<owner>/<repo>/settings/actions
# where you only need to set "Read repository contents and packages permissions"
#
# otherwise you'll get an opaque 403 on the pull-request workflow
permissions:
```